product-service/test/handlers/post-products.test.js

```diff
@@ -0,0 +1,80 @@
+import * as validator from '../../src/utils/product-schema-validator';
+import * as productRepository from '../../src/repositories/product-repository';
+import * as response from '../../src/services/response';
+import { postProducts } from '../../src/handlers/post-products';
```

Comment:
would be nice to add aliases as `@/services` or `@services` instead of `../../../`